source/library/Monadoc/Model/Blob.hs

```diff
@@ -49,9 +49,9 @@ migrations =
   ]
 
 new :: ByteString.ByteString -> Blob
-new contents =
+new c =
```

Comment:
This seems like perhaps a GHC bug. The `contents` argument should be unambiguous here.

Reply:
It's a different module, but the warning I'm seeing is here: https://github.com/tfausak/monadoc/actions/runs/7648996583/job/20842674703#step:18:74

Reply:
Reported upstream: https://gitlab.haskell.org/ghc/ghc/-/issues/24381